template/app/src/main/java/com/xmartlabs/template/model/AuthResponse.kt

```diff
@@ -0,0 +1,12 @@
+package com.xmartlabs.template.model
+
+
+import org.parceler.Parcel
+
+//TODO: check service auth response to match app's AuthResponse fields
+@Parcel(Parcel.Serialization.BEAN)
+class AuthResponse {
+  var accessToken: String? = null
```

Comment:
Data classes cannot have a no args constructor, and that will probably be an issue with the serialization stuff we use